eng/pipelines/build-windows-job.yml

```diff
@@ -37,13 +37,13 @@ jobs:
       displayName: Restore
       inputs:
         filePath: eng/build.ps1
-        arguments: -configuration ${{ parameters.configuration }} -prepareMachine -ci -restore -binaryLog
+        arguments: -configuration ${{ parameters.configuration }} -prepareMachine -ci -restore -binaryLog /p:UsingToolPdbConverter=false /p:UsingToolVSSDK=false  /p:GenerateSatelliteAssemblies=false
```

Comment:
```suggestion
        arguments: -configuration ${{ parameters.configuration }} -prepareMachine -ci -restore -binaryLog /p:UsingToolPdbConverter=false /p:UsingToolVSSDK=false /p:GenerateSatelliteAssemblies=false
```
nit

Reply:
Do we need to duplicate these new arguments across restore and build? I can see that the build also has 'PublishReadyToRun=false'. How do we know which ones need to be duplicated/shared or not?